setup.py

```diff
@@ -38,7 +38,7 @@
     zip_safe=False,
     python_requires=">=3.7",
     install_requires=[
-        "flake8 >= 3.2.1, < 5.0.0",
+        "flake8 >= 3.2.1",
```

Comment:
Should we pin to flake8 < 6.0.0? Until #183 is properly resolved, this is probably going to keep happening with each flake8 major release.

Reply:
It's safer to pin it. I prefer a "better-safe-than-sorry" approach although there are some quite vocal people that disagree.

Reply:
Yeah, I prefer the idea of pinning here, given the situation we're in. I'll make the change.

Reply:
I usually dislike upper bounding, but since we rely on private API, upper bounding here seems wise